apps/Steam/install-64

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+install_packages libnss3:armhf libnm0:armhf libdbus-glib-1-2:armhf libudev1:armhf libnspr4:armhf libgudev-1.0-0:armhf libusb-1.0-0:armhf libappindicator1:armhf libnss3 libnm0 libdbus-glib-1-2 libudev1 libnspr4 libgudev-1.0-0 libusb-1.0-0 libappindicator1 || exit 1
```

Comment:
libappindicator1 isn't available in the debian bullseye repos, but you can install it with the following: 

armhf:

http://ftp.us.debian.org/debian/pool/main/liba/libappindicator/libappindicator1_0.4.92-7_armhf.deb
http://ftp.us.debian.org/debian/pool/main/libi/libindicator/libindicator7_0.5.0-4_armhf.deb

arm64:

http://ftp.us.debian.org/debian/pool/main/liba/libappindicator/libappindicator1_0.4.92-7_arm64.deb
http://ftp.us.debian.org/debian/pool/main/libi/libindicator/libindicator7_0.5.0-4_arm64.deb

Reply:
yeah debian really needs to fix their crap with this...
debian doesn't have support for GTK3 in their libappindicator1 source

ubuntu diverged debian in this package to add GTK3 support in 2012 and debian has never adopted it

they have their own reasons for doing so, debian removed the packages and now everything is supposed to use `libayatana-appindicator`
see the announcement here: https://lists.debian.org/debian-devel/2018/03/msg00506.html

Reply:
the question is... does steam support `libayatana-appindicator` instead of the deprecated `libappindicator`

Reply:
@ryanfortner my suggestion would be just make `libappindicator` an optional dependency
it probably doesn't work on bullseye's GTK3 based DE anyway

Reply:
> my suggestion would be just make libappindicator an optional dependency
it probably doesn't work on bullseye's GTK3 based DE anyway

I think we should be safe and install it anyway. After all, it doesn't conflict with other packages we're installing here.

Reply:
I will check this with steam deb's control file and debian & ubuntu repos and make modifications accordingly

Reply:
I already checked, neither the debian or ubuntu repos include `libappindicator1` or `libayatana-appindicator` at all
usually its a nice to have, not required

Reply:
> this is for showing the steam indicator in the top bar

Are you referring to the system tray?

Reply:
yes, top bar, system tray, taskbar. whatever you want to call it.

Reply:
@ryanfortner can you try it removing the libappindicator1:armhf and libappindicator1 from install_packages 
it should probably work as it is not a dependency in either steam deb or steam libs deb

Reply:
> @ryanfortner can you try it removing the libappindicator1:armhf and libappindicator1 from install_packages it should probably work as it is not a dependency in either steam deb or steam libs deb

@Jai-JAP if it's not a dependency then why is it being installed in this script?

Reply:
This dependencies are same as in install-32 also copied for arm64